src/api/v3/feedback/page_feedback.py

```diff
@@ -23,32 +24,43 @@ class FeedbackData:
     :type emotion: int, optional
     """
 
-    def __init__(self, page_id, permalink, comment, emotion):
+    def __init__(self, page_id, permalink, comment, rating, category):
         self.page_id = page_id
         self.permalink = permalink
         self.comment = comment
-        self.emotion = emotion
+        self.category = category
+        self.rating = rating
 
     @classmethod
     def from_dict(cls, feedback_dict):
```

Comment:
@mael-fosso @emmakemb @AminAlAit as Timo mentioned, we are currently not using the JSON object but HTTP POST variables. So we'd need a second method that reads the POST variables and turns them into the class properties. Does one of you want to take care? Basically its a function `def from_post()` with `request.POST.get("permalink", "")`